### DISCUSSION TITLE
Use Blazored modal from rows of table

### DISCUSSION THREAD

**Discussion by holguinsoft** [Blazored/Modal#401] [Q&A]
Hello.

I have started using this plugin and it is really very good.
I wanted to see how I could use it from rows of a table

I leave a piece of code of what I like to do               

```cshtml
 @foreach (var file in DataSource.SiteFiles!) {
    <tr>
    
        <td>
            <div class="form-check">
              <input class="form-check-input" type="checkbox" @bind="@file.IsCurrent" disabled>
            </div>
        </td>
        <td>
    
            <button @onclick="@(()=>Modal.Show<FetchData>("Modal Popup Example"))" class="btn btn-primary">View</button>
    
        </td>
```

Regards,

**Comment by larsk2009** [comment]
You can open the modal from any place, so also from the button in a table row. Is there any specific issue you are seeing?

**Reply by holguinsoft** [reply]
Hi Iarsk

Now I have reviewed the code and I realize that I am calling the modal inside a <form> tag. If I put, I call the Modal outside the form it is shown. Any solution for this?

Regards,

**Reply by larsk2009** [reply]
I don't think I really understand what you are saying. Could you provide a GitHub repository which can reproduce the problem.
